components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -513,14 +760,9 @@ public User updateUser(User user, Map<String, Boolean> requiredAttributes) throw
                 } else {
                     addSCIMUsers(filteredUsers, userNames, requiredClaimsInLocalDialect, scimToLocalClaimsMap);
                 }
-                log.info("Users filtered through SCIM for the filter: " + attributeName + filterOperation +
-                        attributeValue);
+            } catch (UserStoreException e) {
+                throw new CharonException("Error in retrieve user details. ", e);
```

Comment:
Add user identifier to identify the failed user

Reply:
Since user identifier is an array of username skip to add the identifier